source/Octopus.Client/IOctopusAsyncRepository.cs

```diff
@@ -1,68 +1,82 @@
         IDeploymentProcessRepository DeploymentProcesses { get; }
         IDeploymentRepository Deployments { get; }
         IEnvironmentRepository Environments { get; }
-        IEventRepository Events { get; }
-        IFeaturesConfigurationRepository FeaturesConfiguration { get; }
         IFeedRepository Feeds { get; }
         IInterruptionRepository Interruptions { get; }
         ILibraryVariableSetRepository LibraryVariableSets { get; }
         ILifecyclesRepository Lifecycles { get; }
         IMachinePolicyRepository MachinePolicies { get; }
         IMachineRepository Machines { get; }
         IMachineRoleRepository MachineRoles { get; }
-        IMigrationRepository Migrations { get; }
-        IOctopusServerNodeRepository OctopusServerNodes { get; }
-        IPerformanceConfigurationRepository PerformanceConfiguration { get; }
         IProjectGroupRepository ProjectGroups { get; }
         IProjectRepository Projects { get; }
         IProjectTriggerRepository ProjectTriggers { get; }
         IProxyRepository Proxies { get; }
         IReleaseRepository Releases { get; }
         IRetentionPolicyRepository RetentionPolicies { get; }
-        ISchedulerRepository Schedulers { get; }
-        IServerStatusRepository ServerStatus { get; }
-        ISpaceRepository Spaces { get; }
         ISubscriptionRepository Subscriptions { get; }
         ITagSetRepository TagSets { get; }
-        ITaskRepository Tasks { get; }
-        ITeamsRepository Teams { get; }
         ITenantRepository Tenants { get; }
         ITenantVariablesRepository TenantVariables { get; }
-        IUserRepository Users { get; }
-        IUserRolesRepository UserRoles { get; }
         IVariableSetRepository VariableSets { get; }
         IWorkerPoolRepository WorkerPools { get; }
         IWorkerRepository Workers { get; }
+        SpaceRootResource SpaceRootDocument { get; }
+    }
+
+    public interface ISystemScopedRepository
+    {
+        ISchedulerRepository Schedulers { get; }
+        IServerStatusRepository ServerStatus { get; }
+        ISpaceRepository Spaces { get; }
+        IUserRepository Users { get; }
+        IUserRolesRepository UserRoles { get; }
+        IBackupRepository Backups { get; }
+        IMigrationRepository Migrations { get; }
+        IOctopusServerNodeRepository OctopusServerNodes { get; }
+        IPerformanceConfigurationRepository PerformanceConfiguration { get; }
+        IConfigurationRepository Configuration { get; }
+        ICommunityActionTemplateRepository CommunityActionTemplates { get; }
+        IFeaturesConfigurationRepository FeaturesConfiguration { get; }
+    }
+
+    /// <summary>
+    /// A simplified interface to commonly-used parts of the API.
+    /// Functionality not exposed by this interface can be accessed
+    /// using <see cref="IOctopusAsyncRepository.Client" />.
+    /// </summary>
+    public interface IOctopusAsyncRepository : ISpaceScopedRepository, ISystemScopedRepository
```

Comment:
To make it backward compatible, `IOctopusAsyncRepository` as the root will implement both `ISpaceScopedRepository`, `ISystemScopedRepository`. Mixed scope repositories can be moved out to its own type